.github/workflows/tests.yaml

```diff
@@ -45,4 +45,4 @@ jobs:
         flags: unittests
         env_vars: OS,PYTHON
         name: codecov-umbrella
-        fail_ci_if_error: true
\ No newline at end of file
+        fail_ci_if_error: true
```

Comment:
I'm not actually sure how these lines have changed.  Also, in general this file is something that I'd love to learn more about.  I gather it's the automatic test script for the CICD.

Reply:
it's a newline change.

In general, you always finish a file with a newline. This probably dates back to when computers were less reliable and it was common to have your data terminate suddenly. In this case, there's no easy way to tell if the file has had an issue if it suddenly stops, but if it finishes on a blank line, the chances that it's lost data is minimal.

I just copy pasted this file from the bayesnets repo which itself was copy pasted from the github help. It's a few hours of copy pasting and trying things out, I don't have any special insight lol